storey/dtypes.py

```diff
@@ -288,8 +288,6 @@ def __init__(self, name: str, field: Union[str, Callable[[Event], object]], aggr
             self.value_extractor = field
         elif isinstance(field, str):
             self.value_extractor = lambda element: element[field]
-        else:
```

Comment:
Why did you removed this check altogether?

Reply:
when i'm creating FieldAggregators (line 200) i create them with None column field cause i don't have one and it doesn't matter in query step
i can change it to a random string or whatever

Reply:
I would allow None, but still check that no other weird type is passed